Whisky/Extensions/View+Extension.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  View+Extension.swift
+//  Whisky
+//
+//  This file is part of Whisky.
+//
+//  Whisky is free software: you can redistribute it and/or modify it under the terms
+//  of the GNU General Public License as published by the Free Software Foundation,
+//  either version 3 of the License, or (at your option) any later version.
+//
+//  Whisky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+//  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+//  See the GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License along with Whisky.
+//  If not, see https://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+extension View {
+    func fixedHorizontalSize() -> some View {
```

Comment:
I'm not sure these extensions are necessary

Reply:
Although this extension is not strictly necessary, I believe it's quite useful. It is more concise than the original, making it easier to read and less prone to errors. Additionally, I deleted unused `fixedHorizontalSize()` function.